worker-document/src/main/java/com/hpe/caf/worker/document/scripting/specs/InlineScriptSpec.java

```diff
@@ -69,6 +72,12 @@ public boolean isStatic()
         return true;
     }
 
+    @Override
+    public ScriptEngineType getEngineType()
+    {
+        return engineType;
+    }
+
     @Override
     protected void setScriptSpecField(final DocumentWorkerScript script)
```

Comment:
None of the `setScriptSpecField()` methods have been updated.  Is the engine getting set on the output messages, from the workflow worker for example?  If so then how?  I would have expected the `setScriptSpecField()` methods to be updated to do this.

Reply:
Actually on reflection it might be better to do it in the `AbstractScriptSpec::toDocumentWorkerScript` method, but regardless it's still not done.